### ISSUE TITLE
User Experience

### ISSUE THREAD

**Issue by BrainonSilicon** [alan-turing-institute/AutisticaCitizenScience#573]
# Summary

> 🌟 **This issue outlines the design process steps that need to happen in order to develop a "User Experience" for the platform.**  

The "user experience" of the platform covers everything from the aesthetic (the look) to the usability and functionality of the platform. It's really important to follow a process that has the user's needs first in order to develop a platform that's accessible. 

As designers, we have tounderstand what a user, or a group of users (aka. persona) needs in order to achieve their goal (aka. why they came to a page). "User" means anyone who will use the platform. The users could be researchers, autistic individuals, and policy makers. These are examples of "personas", or "people who will use the platform". 

You can read more about the UX Design process [here](https://xd.adobe.com/ideas/guides/ux-design-process-steps/)

### 1. Defining the Space
- what it is
- who will use it
- why they will use it
We know what the platform is, but we need a better understanding of who will use it, and why. 

We've got a starting mock up for the platform (found [here](https://www.figma.com/file/HxqTSdeyAUNMualnBV0PET/Autspace-(WIP)) on Figma) but need to dig deeper into why and who in order to develop something great!

### 2. Research
- understanding who the users are
- understanding what the needs of different users are
- understanding why they are there (and what they're looking for)

We've summarized the feedback and focus group research we've done  into a spreadsheet here and in #560 
There is also a list of requirements for the design based on accessibility research and user needs in the [platform-design](https://github.com/alan-turing-institute/AutSPACEs/tree/main/platform-design) folder of the AutSPACEs repo.

We also have some user stories under the issues labeled [user-story](https://github.com/alan-turing-institute/AutisticaCitizenScience/labels/user-story)

- [ ] Next step is to identify the core needs of a persona for each page (using user stories) and to identify the "value propositions" that need to be in each area of the platform. This can be done through story boarding and using the research that's been done so far. 

### 3. Design

Design has many steps. To start with, let's focus on the steps below:

#### 3a. Defining the Information Architecture 
- Ensuring the user finds what they're looking for 

IA informs almost everything else that will go onto a page of the platform (e.g. the content strategy, user interface design, and interaction design). Read more about IA [here](https://xd.adobe.com/ideas/process/information-architecture/) 

We've captured some of the IA in a User Flow [here](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/platform-designs/user-flow)

#### 3b. Wireframes and prototypes 
- [ ] We need to develop grid matched wireframes for each page that contain the features that will address the user needs. 

#### 3c. Design Systems
The design system has been captured in other issues. 

## Suggestions for tasks that can fix this issue

* [ ] Who will be using the page and why (what are their goals and what do they need to achieve them?)
* [ ] What pages need to exist and why (what do they achieve and how do they address a user need?)
* [ ] What functional components and values props need to go on each page?
* [ ] Develop wireframes following grid format that are matched with accessibilty guidelines for each page
* [ ] Create copy for each component

## Who can help

For the research
- anyone involved with the project! As users of the platform, we want to work together to design it so everyone can use it. 

For the design 
- anyone with knowledge of UX design can help!

**Comment by BrainonSilicon** [comment]
Hey @GeorgiaHCA and @soledadli. 

I wrote the issue above to outline some of the steps we need to do in order to develop the platform's design. After going through some of the collated research and user flow it seems like we've jumped ahead and are a bit blocked by not having user needs and personas defined. 

I've outlined a few actionable steps the design team can tackle in order to move forward with the design process and I've also created the `User Experience` label so we can track different pieces of work as they progress.

**Comment by soledadli** [comment]
Here is the link of the [User Stories](https://docs.google.com/presentation/d/1SCBVKfA6l2OlOuFOR2xKyU9804OpHr1DCq607bMJ2P8/edit#slide=id.gfcfbc847d0_0_132) created for the AutSPACEs platform. Please feel free to leave your comments and notes here. It´s still ongoing.

**Comment by soledadli** [comment]
Here is[ the link of the Information Architecture](https://drive.google.com/file/d/1THiziivGBr89J5x9oJahXuHtL7iib8J1/view?usp=sharing) for the AutSPACEs Figma Prototype.

**Comment by soledadli** [comment]
Update for User Persona:
* We presented the persona at AutSPACEs meet-up on Nov 11, 2021. 
* As suggested, a new persona for **the carer** will be created. 
* Here is the [Persona Feedback](https://docs.google.com/document/d/1W9Pk5DfD9nWuwbwiok7DnMVCQdMzbRYHKsYgU8Iqoew/edit#) for you to contribute your advice.

**Comment by skfantoni** [comment]
Hi, @BrainonSilicon I was also thinking of going through a similar process to the one you describe under 'Research' to develop a better understanding of users and their potential motives for using the platform. My intention was to do this to help improve the CoC. 

If I go through the personas @soledadli has written and compare them to the notes from @GeorgiaHCA's summaries, to collate a deeper understanding of needs would that be useful to you too?

**Comment by BrainonSilicon** [comment]
Hi @skfantoni - that sounds awesome. 

I think there may be a super useful way to integrate the resources together so that they can be useful for your CoC iterations as well as other components for the platform. 

## For user needs 
- the feedback summaries and research/focus group folders are great resources 
- the personas are then a solid foundation for identifying the user groups and their potential needs
- we've been building out a User Needs matrix (not yet on github as it's been blocked by a few tasks) which you can see [here](https://docs.google.com/spreadsheets/d/11vbZ0TYvY2qq-EgF5IHNowC4Mfksf_fekJ5rUXY4deM/edit?usp=sharing). The second page `Draft` has the most detailed section. 
- @soledadli has been creating user flows which may also be helpful. Soleded, can you direct @skfantoni towards where those are?

**Comment by skfantoni** [comment]
Cool @BrainonSilicon, I'll start going through the summaries tomorrow and see what I can collate. Then we can make some comparisons and see what's missing.

**Comment by soledadli** [comment]
@skfantoni Here are the diagrams for user flows. I am focusing on moderators these days. Let me know whether you have any feedback for me. 🌟 
* [Moderation Workflow](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&page-id=dlBrGaaeTGBov1gFk8MI&title=user_flow_autspacs.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1pohsA28S36n1yIWvAJhNd5k79qazjSeg%26export%3Ddownload)
* Moderator User Flow:
   * [Moderate New Experiences](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&page-id=dlBrGaaeTGBov1gFk8MI&title=user_flow_autspacs.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1pohsA28S36n1yIWvAJhNd5k79qazjSeg%26export%3Ddownload)
  * [Review Moderated Experiences](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&page-id=pQMC_svEnV-OpvSRIs3h&title=user_flow_autspacs.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1pohsA28S36n1yIWvAJhNd5k79qazjSeg%26export%3Ddownload)

**Comment by skfantoni** [comment]
@BrainonSilicon @soledadli So I think I've come to the conclusion that there are essentially three separate personas for autistic people: 

1. Those who want a safe place to write about their experiences
2. Those who want guidance from other autistic people
3. Those who simply want to contribute to research

I think it's important in regards to design and user flow that each of these is taken into consideration separately.

**Comment by skfantoni** [comment]
@soledadli I don't want to step on your toes, so would it be ok with you if I were to take the current personas and use them as a template to make some new, more specific ones?

**Comment by soledadli** [comment]
> @soledadli I don't want to step on your toes, so would it be ok with you if I were to take the current personas and use them as a template to make some new, more specific ones?

@skfantoni No stepping toes at all. 🌟 It is important and necessary to expand the user personas and let them grow with the project. I am looking forward to readings your personas! Here is the [persona template](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/platform-designs/design-research/Personas/Persona-Template.md).

**Comment by soledadli** [comment]
> @BrainonSilicon @soledadli So I think I've come to the conclusion that there are essentially three separate personas for autistic people:
> 
> 1. Those who want a safe place to write about their experiences
> 2. Those who want guidance from other autistic people
> 3. Those who simply want to contribute to research
> 
> I think it's important in regards to design and user flow that each of these is taken into consideration separately.
> 
> For reference, [these](https://docs.google.com/document/d/1KC8qahyWL8OjeHoj7LlKkYbE_RCRxZQVGd8xxNtcLUU/edit?usp=sharing) are the key points I picked up on when reading through the summaries.

@skfantoni Thanks for identifying the missing parts here. 🌷  I will remember to check the key points and create user flows from these three perspectives. Once the flows are created for autistic people, I will share them out here for feedback and advice from the community. ⭐

**Comment by skfantoni** [comment]
I've completed the new [Autistic Person personas ](https://docs.google.com/document/d/166ATKNiqsSnIUGsOWr2-muMMXNxKYN1So51jJ0DPOvg/edit?usp=sharing). I've added notes in regards to what I think needs to be included for each persona. It would be great to know whether the Dev team think these ideas are possible. I will discuss it at the meet-up on 23rd December.

**Comment by GeorgiaHCA** [comment]
Fabulous, thank you @skfantoni! Looking forward to discussing them :)

**Comment by GeorgiaHCA** [comment]
Thanks for the presentation @skfantoni ! Looking forward to 2022 and excited for your input too Sue

**Comment by BrainonSilicon** [comment]
Linking this to PR #637 which is about the copy for the website.
